os-eap-activemq-rar/module.yaml

```diff
@@ -7,5 +7,5 @@ execute:
   user: '185'
 
 artifacts:
-- url: https://maven.repository.redhat.com/ga/org/apache/activemq/activemq-rar/5.11.0.redhat-630329/activemq-rar-5.11.0.redhat-630329.rar
+- url: https://maven.repository.redhat.com/ga/org/apache/activemq/activemq-rar/5.11.0.redhat-630347/activemq-rar-5.11.0.redhat-630347.rar
   md5: 76ed338d86bb3dbba2d4bd48daff7dbb
```

Comment:
@luck3y shouldn't be md5 checksum updated also?

Reply:
Yep, sorry, forgot to actually push that bit. Fixed, thanks!